.tekton/fact-component-pipeline.yaml

```
@@ -243,11 +243,22 @@ spec:
     - name: git-basic-auth
       workspace: git-auth
 
-  # TODO: switch this task to matrix and add ppc64le, s390x and arm64 builds
-  - name: build-container-amd64
+  - name: build-containers
+    matrix:
+      include:
+      - name: amd64
+        params:
+          - name: PLATFORM
+            value: linux/amd64
+          - name: IMAGE
+            value: $(params.output-image-repo):$(tasks.determine-image-tag.results.IMAGE_TAG)-amd64
+      - name: arm64
+        params:
+          - name: PLATFORM
+            value: linux/arm64
```

You may try pick one of CPU-optimized types to see if this can make builds faster.
See https://konflux.pages.redhat.com/docs/users/getting-started/multi-platform-builds.html#arm64-2
E.g.
```suggestion
      - name: arm64
        params:
          - name: PLATFORM
            value: linux-cxlarge/arm64
```